src/mqueryfront/src/QueryStatus.js

```diff
@@ -265,6 +265,25 @@ class QueryStatus extends Component {
         if (this.state.job.status === "expired") {
             return ReturnExpiredJob(this.state.job.error);
         }
+
+        let results;
```

Comment:
should this be `<div />` by default? I'm not sure how adding `undefined` element will work